app/javascript/components/StudioAssessmentQuestion/index.js

```diff
@@ -0,0 +1,285 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik, Form, Field } from 'formik';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { apiPost, apiPatch } from 'utils/axios';
+import { Button } from '@material-ui/core';
+// import { consoleSandbox } from '@sentry/utils';
+import RadioButtonsGroup from './radioButtons';
+import styles from './styles';
+
+const questions = [
+  'Understands the Big Picture of the Full Stack',
+  'JavaScript Fundamentals',
+  'Understands Version Control',
+  'React Core Competencies',
+  'NodeJs Core Competencies',
+  'Database Core Competencies',
+  'Problem Solving: Whiteboarding',
+];
+
+const questionContent = [
```

Comment:
can we move these to a separate file and import them into this one, also will  you update these with the correct questions from the rubric